cluster/gce/util.sh

```diff
@@ -156,8 +156,40 @@ function copy-if-not-staged() {
   fi
 }
 
+# Prepare a tarball of kube-system manifests for trusty based cluster.
+#
+# Vars set:
+#   KUBE_MANIFESTS_TAR_URL
+#   KUBE_MANIFESTS_TAR_HASH
+function prepare-manifests-tar() {
```

Comment:
Yep, just wanted to make sure you weren't caught offguard by this working fine in CI and then failing in the next release. It'd be worth testing out the next 1.2 alpha release (they should get cut every 2 weeks or so) once both of your PRs are in to make sure the files are in the right place in the release.